presto-docs/src/main/sphinx/release/release-0.234.rst

```diff
@@ -0,0 +1,48 @@
+=============
+Release 0.234
+=============
+
+General Changes
+_______________
+* Fix query failure for cross-joining bucketed tables with ``COALESCE`` when the bucket counts are compatible but mismatched.
+* Fix query failure for window functions with ``IGNORE NULLS``.
+* Fix a regression in the new scheduler temporarily by setting the default for the configuration property ``use-legacy-scheduler`` to ``true``.
+* Improve system reliability by preventing additional queries to execute when the total number of tasks exceeds the limit, which can be
```

Comment:
Lets use the format in https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#phrasing.

```
Improve coordinator reliability by adding support to cap the number of total tasks running in a coordinator by pausing scheduling. This can be enabled by setting the ......
```